app/uk/gov/hmrc/trackingconsentfrontend/assets/test/domain/userPreference.spec.ts

```diff
@@ -195,9 +235,9 @@ describe('User Preference Factory', () => {
       expect(new Date().getTime()).toEqual(123)
     })
     it('cookies should be settable and gettable', () => {
-      expect(Cookies.get('abcdef')).toBeUndefined()
+      expect(Cookies.getJSON('abcdef')).toBeUndefined()
       Cookies.set('abcdef', 'this is my cookie value')
-      expect(Cookies.get('abcdef')).toBe('this is my cookie value')
+      expect(Cookies.getJSON('abcdef')).toBe('this is my cookie value')
```

Comment:
This is to prove that the mock behaves correctly, it would be good to show it working with an object.